config/locales/user_mailer/en.yml

```diff
@@ -209,7 +209,8 @@ en:
         help, please visit the %{app_name_html} %{help_link_html} or
         %{contact_link_html}.
       info_html: '<p>Your %{app_name} account was just used to sign in on a new
-        device.</p><br/><p><strong>%{date}<br/>%{location}</strong></p>'
```

Comment:
If we have the energy, I'd recommend moving a lot of this markup into the ERB template and simplifying these strings